build/release/release.properties

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+
+# This only needs to be non-empty to stop outputting debug messages. 
+DEBUG="false"
+
+# Bintray
+# - Release 
+# - Snapshots
+BINTRAY_REPO=
```

Comment:
when does this get set?

Reply:
actually - we don't use that anymore.  it's set in release.sh as part of type.  let me clean that up.